### ISSUE TITLE
bug in Exploring_Data.ipynb for holoviews1.4.1 and matplotlib 1.5.0 and Python 3.4

### ISSUE THREAD

**Issue by stonebig** [ioam/holoviews#386]
I get this error in `Exploring_Data.ipynb`:
````
surface_wind.info
````

````
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-b44694d72f09> in <module>()
----> 1 surface_wind.info

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\holoviews\core\ndmapping.py in info(self)
    425                 dmin, dmax = self.range(d.name)
    426                 if d.value_format:
--> 427                     dmin, dmax = d.value_format(dmin), d.value_format(dmax)
    428                 info_str += '\t %s: %s...%s \n' % (str(d), dmin, dmax)
    429         print(info_str)

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\matplotlib\dates.py in __call__(self, x, pos)
    483                              'plotting dates, e.g., with ax.xaxis_date()')
    484         dt = num2date(x, self.tz)
--> 485         return self.strftime(dt, self.fmt)
    486 
    487     def set_tzinfo(self, tz):

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\matplotlib\dates.py in strftime(self, dt, fmt)
    587             # Note: in python 3.3 this is okay for years >= 1000,
    588             # refer to http://bugs.python.org/issue177742>
--> 589             return cbook.unicode_safe(dt.strftime(fmt))
    590 
    591         return self.strftime_pre_1900(dt, fmt)

ValueError: Invalid format string
````

the error keeps coming in cells after (french datetime notation on my windows7 )

**Comment by philippjfr** [comment]
As far as I know this bug has been fixed f7670ac9e76bbfd274e5437f5eb1bb99cbd80e39. Is that correct? Can I close?

**Comment by jbednar** [comment]
Yes, I think it can close -- @stonebig suggested the change, and I checked that it worked fine at my end, then committed to the main repository.  I'm not sure that my message on commit f7670ac is correct, i.e. maybe it's due to his non-English locale rather than Python3 per se, but in either case it seems to fix it.  I'll close now, but if there is any issue it can be reopened.
